### DISCUSSION TITLE
Mac Docker LOCAL option for external storage

### DISCUSSION THREAD

**Discussion by swilson969** [nextcloud/all-in-one#2116] [Questions] [answered]
The readme is excellent, yet sadly with AIO running great on Docker Desktop Mac, I can't make the final leap of enabling the LOCAL option in external storage (not listed, even after restart using the AIO interface post App addition) to point to a host directory (eg /Volumes/myStuff) as the readme lacks an explicit set of instructions for a Mac host as there are for a Linux host.  I scoured the net, and so my humble request: point me to a previous answer or tutorial (which must exist), and thanks in advance!!

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one#how-to-allow-the-nextcloud-container-to-access-directories-on-the-host

**Reply by swilson969** [reply]
Thanks Simon.  Chalk it up to me being overcautious/nooBish, seemed to easy to be true.

For community service, below is my startup for a MacOS Ventura hosted AIO on docker install that accomplishes what I was after (for other nooBs, note that there are only two differences from that which is shown in the readme for Linux - nothing to fear it's EASY!!)

```
# For MacOSX hosts without a web server or reverse proxy (like Apache, Nginx and else) already in place:
sudo docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock.raw:/var/run/docker.sock:ro \ # notice there's a raw in there for Mac
-e NEXTCLOUD_MOUNT="/Volumes/your_drive/" \ # file path to your drive on your Mac host that you'll enable via external storage app 
nextcloud/all-in-one:latest
```
